.vsts-dotnet-ci.yml

```diff
@@ -0,0 +1,136 @@
+trigger:
+- vs15.9
+
+pr:
+- vs15.9
+
+jobs:
+- job: innerloop_OSX1013_CoreCLR
+  displayName: OSX10.13 Build for CoreCLR
+  pool:
+    vmImage: macOS-10.13
```

Comment:
nit: would the "Hosted macOS High Sierra" pool work for this?  It would be nice to move to hosted pools where possible

Reply:
This is the hosted pool.  They wanted us to use this format (the other format is deprecated)

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops#use-a-microsoft-hosted-agent

Reply:
boom